latch_cli/extras/nextflow/tasks/process.py

```diff
@@ -161,6 +161,39 @@ def get_fn_code(self, nf_script_path_in_container: Path):
             run_task_entrypoint[2] = str(Path("/root") / stem)
             run_task_entrypoint.extend(["-entry", self.calling_subwf_name])
 
+        if self.wf._docker_metadata is not None:
```

Comment:
add a todo to check if the process has a container directive? so we don't always login

Reply:
+1